jbrowse/test/src/org/labkey/test/tests/external/labModules/JBrowseTest.java

```diff
@@ -67,18 +77,33 @@ protected String getProjectName()
     public void testSteps() throws Exception
     {
         setUpTest();
-
+        testDemoNoSession();
         testDemoUi();
         //testOutputFileProcessing();
     }
 
-    private void testDemoUi()
+    private void testDemoNoSession()
     {
-        beginAt("/home/jbrowse-jbrowse.view");
-
-        waitAndClick(Locator.tagWithClass("span", "MuiButton-label"));
+        beginAt("/home/jbrowse-jbrowse.view?");
+        assertElementVisible(Locator.xpath("/html/body/div[2]/div/div[3]/div/div/div/p"));
```

Comment:
can you make this a more specific xpath? like a tag + class + text or something?

also, it is generally better to do a waitForElement(). assertElementVisible() might have some kind of wait build into it, but explicitly waiting for some specific element makes your tests less prone to breaking from timing issues.

Reply:
Updated